contacts.py

```diff
@@ -555,54 +555,57 @@ def send_status(self, data, args):
         # These are fields allowed in the send_status, and just copied from top level into each data point
         # Note memo is not supported yet and is a placeholder
         repeated_fields = {k: data.get(k) for k in ['memo', 'replaces', 'status'] if data.get(k)}
-        return self.send_or_sync(data, repeated_fields)
+        self.send_or_sync(data, repeated_fields)
+        return {"status": "ok"}
 
     # Common part of both /status/send and sync reception
-    def send_or_sync(self, data, repeated_fields):
-        floating_seconds = current_time()
+    def send_or_sync(self, data, repeated_fields=None, floating_seconds=None):
+        if not floating_seconds:
+            floating_seconds= current_time()
         serial_number = 0
         # first process contacts, then process geocode
         for contact in data.get('contact_ids', []):
-            contact.update(repeated_fields)
+            contact.update(repeated_fields or {})
```

Comment:
why not just default repeated_fields to {} then you don't have to have this 'or'

Reply:
Because if you do `repeated_fields={}` in the args, and then change repeated_fields (e.g. in contact.update), it changes {} so the next time it breaks. Its `x={}` or `x=[]` in the function header is a known anti-pattern that my IDE reminded me of (I remember it from python days as well, getting tripped up by it a couple of times in obscure ways)

Reply:
i did not know that.  thanks... well you can do {}.copy(), but that's horrible :-)

Reply:
Or you can do the "or" construct which is what I usually do ....